### PR TITLE
fix audit certificates upload in IE

### PR DESCRIPTION
**Pull request by dreamfall**

### CODE REVIEW

File: app/controllers/users/audit_certificates_controller.rb [hunk 1]
```diff
@@ -62,26 +70,24 @@ def destroy
...
-      end
-
-      params.require(:audit_certificate).permit(
-        :attachment
-      )
+  def audit_certificate_params
+    # This is fix of "missing 'audit_certificate' param"
+    # if no any was selected in file input
+    if params[:audit_certificate].blank?
+      params.merge!({
```
Comment (matthewford) [inline app/controllers/users/audit_certificates_controller.rb]: Redundant curly braces around a hash parameter.

File: app/controllers/users/audit_certificates_controller.rb [hunk 2]
```diff
@@ -62,26 +70,24 @@ def destroy
...
+        audit_certificate: {
```
Comment (matthewford) [inline app/controllers/users/audit_certificates_controller.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: app/controllers/users/audit_certificates_controller.rb [hunk 3]
```diff
@@ -62,26 +70,24 @@ def destroy
...
+          attachment: ""
+        }
+      })
```
Comment (matthewford) [inline app/controllers/users/audit_certificates_controller.rb]: Indent the right brace the same as the first position after the preceding left parenthesis.

**Comment by RuslanHamidullin** [comment]
Looks good :+1:

**Pull request merged**